01_01_Basic_Rules.txt

```diff
@@ -628,6 +626,33 @@ Class Feats:
 specifies a class feat. Characters can not take non-class feats instead of class feats 
 or vice-versa. Class Feats are listed in the Classes document.
 
+==============================
+Items and Buying Items
+==============================
+
+	When you build a character, you will get some starting money and the time to
+spend it to acrue the kit that you'll start playing with. But this likely won't
```

Comment:
I'd replace "But" with "However," for a more structurally complete sentence. (The "but" sticks out in a bad way here imo)

Reply:
ya, I really shoudn't start a sentence with "but".